docs/deploy-cluster.md

```diff
@@ -0,0 +1,150 @@
+
+Using `data_path` to set `Meta` and `Storage`'s underlying storage directory.
+
+***
+
+#### StartUp Nebula Cluster
+
+Currently, we have support `scripts/services.sh` to management the nebula cluster.
+
+You can `start`, `stop` and `restart` the cluster using this script.
+
+It's looks like the following command:
+
+```
+scripts/services.sh <start|stop|restart|status|kill>
+```
+
+The metas, storages and graphs contain the host of them.
+
+***
+
+#### Config reference
+
+**Meta Service** support the following config properties.
+
+Property Name               | Default Value            | Description
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon listening port.
+`reuse_port`                | true                     | Whether to turn on the SO_REUSEPORT option.
+`data_path`                 | ""                       | Root data path.
+`peers`                     | ""                       | It is a list of IPs split by comma, the ips number equals replica number. If empty, it means replica is 1.
+`local_ip`                  | ""                       | Local ip speicified for NetworkUtils::getLocalIP.
+`num_io_threads`            | 16                       | Number of IO threads.
+`meta_http_thread_num`      | 3                        | Number of meta daemon's http thread.
+`num_worker_threads`        | 32                       | Number of workers.
+`part_man_type`             | memory                   | memory, meta.
+`pid_file`                  | "pids/nebula-metad.pid"  | File to hold the process id.
+`daemonize`                 | true                     | Whether run as a daemon process.
+`cluster_id`                | 0                        | A unique id for each cluster.
+`putTryNum`                 | 10                       | Try num of store clusterId to kvstore.
+`load_config_interval_secs` | 2 * 60                   | Load config interval.
+`meta_ingest_thread_num`    | 3.                       | Meta daemon's ingest thread number.
+
+**Storage Service** support the following config properties.
+
+Property Name                       | Default Value              | Description
+----------------------------------- | -------------------------- | -----------
+`port`                              | 44500                      | Storage daemon listening port.
+`reuse_port`                        | true                       | Whether to turn on the SO_REUSEPORT option.
+`data_path`                         | ""                         | Root data path, multi paths should be split by comma. For rocksdb engine, one path one instance.
+`local_ip`                          | ""                         | IP address which is used to identify this server, combined with the listen port.
+`mock_server`                       | true                       | Start mock server.
```

Comment:
Do we really need to expose this? What is a "mock_server"?

Reply:
I found this flag have been removed .